File: WideAreaDecon/Battelle.EPA.WideAreaDecon.API/Enumeration/Job/JobStatus.cs
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Battelle.EPA.WideAreaDecon.API.Enumeration.Job
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum JobStatus
    {
        [EnumMember(Value = "Unknown")]
        Unknown = -3,
        [EnumMember(Value = "Error")]
        Error = -2,
        [EnumMember(Value = "Cancelled")]
        Cancelled,
        [EnumMember(Value = "New")]
        New,
        [EnumMember(Value = "Queued")]
        Queued,
        [EnumMember(Value = "Running")]
        Running,
        [EnumMember(Value = "Completed")]
        Completed,
    }
}

File: WideAreaDecon/Battelle.EPA.WideAreaDecon.Client/src/enums/jobs/jobStatus.ts
// eslint-disable-next-line no-shadow
enum JobStatus {
  unknown = 'Unknown',
  error = 'Error',
  cancelled = 'Cancelled',
  new = 'New',
  queued = 'Queued',
  running = 'Running',
  completed = 'Completed',
}

export default JobStatus;
